jenkins/job-configs/kubernetes-jenkins-pull/global.yaml

```diff
@@ -1,13 +1,28 @@
+- builder:
+    name: ensure-upload-to-gcs-script
+    builders:
+      - shell: |
+          _script_tmp_path='./_tmp/upload-to-gcs.sh'
+          _script_repo_path='hack/jenkins/upload-to-gcs.sh'
+          mkdir -p ./_tmp
+          if [[ "${{ghprbTargetBranch:-}}" == 'master' && -x "./kubernetes/${{_script_repo_path}}" ]]; then
```

Comment:
I don't understand why it needs to be downloaded... don't you already have
the entire repository? You must have downloaded the thing you're about to
test, no?

On Fri, Jun 24, 2016 at 2:02 PM, Jeff Grafton <notifications@github.com>
wrote:

> In jenkins/job-configs/kubernetes-jenkins-pull/global.yaml
> <https://github.com/kubernetes/test-infra/pull/217#discussion_r68462379>:
>
> > @@ -1,13 +1,28 @@
> > +- builder:
> > +    name: ensure-upload-to-gcs-script
> > +    builders:
> > +      - shell: |
> > +          _script_tmp_path='./_tmp/upload-to-gcs.sh'
> > +          _script_repo_path='hack/jenkins/upload-to-gcs.sh'
> > +          mkdir -p ./_tmp
> > +          if [[ "${{ghprbTargetBranch:-}}" == 'master' && -x "./kubernetes/${{_script_repo_path}}" ]]; then
>
> we should probably still optimize post-commit to only download once.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/test-infra/pull/217/files/483c9965c0257087557a0a77551e271db656f504#r68462379>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAnglms9Inx4esltfNjhdT4_4TvZxoJTks5qPEX9gaJpZM4I-E8b>
> .
>